hack/build/crds.bzl

```diff
@@ -43,8 +43,7 @@ def generated_crds(name, go_prefix, paths, out, visibility = [], deps = []):
             "output:crd:dir=$${output_dir};",
             "touch $$out;",
             "for file in $$(find \"$${output_dir}\" -type f | sort -V); do",
-            "  cat \"$$file\" >> \"$$out\";",
-            "  echo \"---\" >> \"$$out\";",
+            "  cat \"$$file\" | sed '/^$$/d' >> \"$$out\";",
```

Comment:
Can you maybe add a comment above this line to describe what the `sed` is for? From what I can tell, it removes double new-lines altogether?

1) does having ^ at the start prevent this line ever concatting together two lines that *shouldn't* be combined?
2) can you add a comment to clarify what this line intends to do - I totally know we'll forget next time it comes to touching this area! 😅

Reply:
This sed command just remove all blank lines, `^$` stands for beginning of the line (^) immediatly followed by end of the line ($), and /d for delete, there are two $ to escape $, so `sed '/^$$/d'`. I'll add a comment to the file ;-)

Reply:
d'oh, yes, of course.. would you believe I even did a class on regex once? Thanks for adding the comment 😄